ci/tasks/create-notification-snippets.yml

```diff
@@ -19,43 +19,29 @@ run:
   args:
     - -c
     - |
-      cat <<EOT >> snippet/start
-      THIS MESSAGE RELATES TO FARGATE - IGNORE UNLESS WORKING ON MIGRATION
-      :rocket: ${ACTION_NAME} of ${APP_NAME} on ${ENV} is beginning
+      APP_RELEASE_NUMBER=$(echo ${APPLICATION_IMAGE_TAG} | sed 's/-release//')
 
-      Version details:
+      cat <<EOT >> snippet/start
+      :rocket: FARGATE ${ACTION_NAME} of <https://github.com/alphagov/pay-${APP_NAME}/releases/tag/alpha_release-${APP_RELEASE_NUMBER}|${APP_NAME} ${APP_RELEASE_NUMBER}-release> on ${ENV} is beginning
```

Comment:
Really nice changes. Maybe something for future thought, I think it might be more useful to link the commit that the release was built from than the release itself since I feel the first question I'm asking if I'm going to look at what's been deployed is, what was the change? e.g. 
https://github.com/alphagov/pay-publicapi/commit/959141185bbd35aa71013cb182f64584b2ca6b98
rather than 
https://github.com/alphagov/pay-publicapi/releases/tag/alpha_release-802

We might not have that information readily available within the job and it's simple to get to the commit from the release page so no biggie.

Reply:
Agreed, that would be useful. I think you're right though, that the commit might not be easily retrievable everywhere yet. I'll add that to the Future Ideas list (shhh, don't call it tech debt... 😉 )